plugins/prometheus.py

```diff
@@ -48,12 +151,21 @@ def get_webhook_key(self):
     def on_receive_webhook(self, url, data, ip, headers):
         json_data = json.loads(data)
         log.info("recv %s", json_data)
-        template = Template(self.store.get("message_template"))
-        for alert in json_data.get("alert", []):
-            for room_id in self.rooms.get_room_ids():
-                log.debug("queued message for room " + room_id + " at " + str(self.queue_counter) + ": %s", alert)
-                queue.put((self.queue_counter, room_id, template.render(alert)))
-                self.queue_counter += 1
+        def process_alerts(alerts, template):
+            for alert in alerts:
+                for room_id in self.rooms.get_room_ids():
+                    try:
+                        if (len(self.rooms.get_content(
+                                room_id, PrometheusPlugin.TYPE_TRACK)["projects"])):
```

Comment:
Your check here is merely checking for a non-zero length array, because you haven't yet implemented the `known_projects` thing. Please can you put some comments explaining:
 - Why you're checking for a non-zero length array
 - What needs to be done in order for it to be actually checking the `projectName`.